vignettes/engines.Rmd

```diff
@@ -0,0 +1,65 @@
+---
```

Comment:
If you render this Rmd I think it is more clear what is going on than trying to read the Rmd directly.